parameters/production-parameters.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
rename this file to production-bounty-dao.json

Reply:
done